modules/consignment-api/ecs.tf

```diff
@@ -0,0 +1,127 @@
+locals {
+  app_port = 8080
+}
+resource "aws_ecs_cluster" "consignment_api_ecs" {
+  name = "${var.app_name}_${var.environment}"
+
+  tags = merge(
+    var.common_tags,
+    map("Name", "consignment_api_${var.environment}")
+  )
+}
+
+data "template_file" "app" {
+  template = file("modules/consignment-api/templates/consignment-api.json.tpl")
+
+  vars = {
+    app_image       = "nationalarchives/sangria:${var.environment}"
```

Comment:
Should the image also be called something like "consignment-api" for consistency?

Reply:
It should, I've changed it.